libraries/src/Router/SiteRouter.php

```diff
@@ -68,7 +68,7 @@ public function __construct(CMSApplication $app = null, AbstractMenu $menu = nul
         $this->menu = $menu ?: $this->app->getMenu();
 
         // Add core rules
-        if ($this->app->get('force_ssl') === 2) {
+        if ($this->app->get('force_ssl') == 2) {
```

Comment:
```suggestion
        if ((int) $this->app->get('force_ssl') === 2) {
```
Maybe it would be more readable like this?

I can confirm by review that at all other places we either do not do a type safe comparison when checking the force_ssl configuration parameter, or we cast it to int when using type safe comparison. Sometimes we do it this way and sometimes that way.

So the PR is ok, and I do not insist on my above suggestion. It might be a matter of taste.